Movies-MVP/Movies-MVP/Core/Builder/Protocols/AssemblyBuilderProtocol.swift

```diff
@@ -0,0 +1,11 @@
+// AssemblyBuilderProtocol.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// AssemblyBuilderProtocol
```

Comment:
```suggestion
/// AssemblyBuilderProtocol
```
Дока не соответсвует